app/services/plum_importer.rb

```diff
@@ -31,6 +31,12 @@ def generate_derivative(member)
       handlers: change_set_persister.handlers
     )
     derivative_change_set_persister.save(change_set: file_change_set)
+  rescue Errno::ENOENT
```

Comment:
It's a little confusing that this method is called generate_derivative but I think from conversation that it actually imports / saves the derivative. It looks like it is saving from a file location; are these applications running on the same box?

Reply:
:+1: I'll fix the name.